src/components/EmojiPicker.vue

```diff
@@ -64,13 +65,21 @@ export default defineComponent({
 </script>
 
 <style lang="scss">
+.emoji-picker.is-dark {
```

Comment:
try to use `.theme--dark {}` and `.theme--light {}`. See example: https://github.com/Adamant-im/adamant-im/blob/6c0b5cba4ec14593494bdc73ab75085069f111fb/src/components/PartnerInfo.vue#L143